config/locales/simple_form.pl.yml

```diff
@@ -4,14 +4,20 @@ pl:
     hints:
       defaults:
         avatar: PNG, GIF lub JPG. Maksymalnie 2MB. Zostanie zmniejszony do 120x120px
-        display_name: Maksymalnie 30 znaków
+        display_name:
+            one: 'Pozostał <span class="name-counter">1</span> znak.'
```

Comment:
the whitespace indent difference between display_name and one/other is 4 spaces instead of 2. this breaks the syntax. see also: same for "note" below